src/plugins/intel_npu/src/al/include/intel_npu/al/config/config.hpp

```diff
@@ -349,9 +381,19 @@ class OptionsDesc final {
     template <class Opt>
     void add();
 
+    template <class Opt>
+    void add(compilerVersion compilerVersionRequirement);
```

Comment:
Seems always use this version of add(), do we still use the one on line 382

Reply:
will remove it, thanks!